include/net/socket.h

```diff
@@ -852,6 +852,9 @@ struct ifreq {
 /** sockopt: Transmission of broadcast messages is supported (ignored, for compatibility) */
```

Comment:
For the commit "Add SO_SNDBUF....", I think this could be merged to the 1st commit as this does not bring much extra info if kept on its own.